TACC/Core-CMS
# docs: asset templates f.k.a. enhance: support template overwrite not inheritance

## Overview

Clean up templates in a way that supports https://github.com/TACC/Core-CMS-Custom and https://github.com/TACC/tup-ui overwrites of them.

## Related

- replaces #487
- replaces #492
- required by https://github.com/TACC/Core-CMS-Resources/pull/189
- required by https://github.com/TACC/Core-CMS-Custom/pull/209

## Testing & UI

> [!IMPORTANT]
> **Skipped**, cuz merging `main` has made changeset just be removing old comments.

1. Deploy all sites that `{% block assets_… %}`.
2. Verify relevant UI (from custom assets is unchanged) i.e. matches prod.

See:
- https://github.com/TACC/Core-CMS-Resources/pull/189
- https://github.com/TACC/Core-CMS-Custom/pull/209

## Changes

- ~~**moved** favicon render from `assets_custom` to `assets_core`~~[^1]
- ~~**added** example in `assets_custom` of how to customize fonts~~[^1]
- **removed** now-unnecessary comments from many templates
- ~~**moved** `page_title` to `base` (so no need to mass duplicate)~~[^1]
- **added** comments explaining `delayed.html` template intended usage

[^1]: Happened in other commits in `main` that have since been merged into this branch.
